packages/terra-consumer-nav/src/components/user-profile/UserProfile.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../variables';
+
+:local {
+  .profile {
+    background: inherit;
+    bottom: 0;
+    height: auto;
+    left: 0;
+    position: fixed;
+    width: inherit;
+  }
+
+  .icon {
+    height: 16px;
+    width: 16px;
+  }
+
+  .link {
+    background: #fff;
+    border: 0;
+    padding: 20px;
+    text-align: left;
+    text-decoration: none;
+    width: 100%;
+
+    &:hover {
+      background: #eee;
+      cursor: pointer;
+    }
+  }
+
+  .link-text {
+    color: #383f42;
+    font-size: 14px;
+    line-height: 18px;
+  }
+
+  .signout-border {
+    border-top: 1px solid #dedfe0;
+    margin: 0;
+    visibility: visible;
+    @media screen and (max-width: $terra-medium-breakpoint) {
+      border-top: 0;
+    }
+  }
+  
+  .nav-item-border {
+    border-bottom: 0;
+    margin: 0;
+
+    @media screen and (max-width: $terra-medium-breakpoint) {
+      border-bottom: 1px solid #dedfe0;
+      margin: 0;
+    }
+  }
+
+  .popup-button {
+    background: inherit;
+    border: 0;
+    cursor: pointer;
+    font-size: 16px;
+    padding: 20px 25px;
+    text-align: center;
+    width: inherit;
+ 
+    @media screen and (max-width: $terra-medium-breakpoint) {
+      color: var(--terra-consumer--mobile-badge-text-color, #fff);
```

Comment:
should not be reusing a variable here, each element should be themed using a different var.